hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/container/upgrade/UpgradeTask.java

```diff
@@ -64,6 +64,9 @@ public class UpgradeTask {
   private final DatanodeStoreSchemaThreeImpl datanodeStoreSchemaThree;
 
   private static final String BACKUP_CONTAINER_DATA_FILE_SUFFIX = ".backup";
+  public static final String MIGRATE_FLAG_FILE_NAME = "migrate";
+  public static final String UPGRADE_LOCK_FILE_NAME = "upgrade.lock";
```

Comment:
Can we use a consistent naming here? 
Say UPGRADE_LOCK_FILE_NAME and UPGRADE_COMPLETE_FILE_NAME